Clarification of acceptable use of "fail" and "disable" (#163)
mongodb/docs-meta: source/style-guide/includes/terms/f.rst

.. glossary::

   fail to

>      :icon-fa4:`times-circle` Use *unable to* instead when referring to an individual or
>      company. 

Reviewer: ```suggestion
     :icon-fa4:`times-circle` Use *unable to* when you refer to an individual or
     company. 
```

>      Use of *fail* is acceptable to describe the performance of a system.

Reviewer: ```suggestion
     You may use *fail* to describe system performance.
```
